src/main/java/htsjdk/tribble/util/HTTPHelper.java

```diff
@@ -90,24 +90,6 @@ public InputStream openInputStream() throws IOException {
         return new WrapperInputStream(connection, connection.getInputStream());
     }
 
-
-    /**
-     * Open an input stream for the requested byte range.  Its the client's responsibility to close the stream.
-     *
-     * @param start start of range in bytes
-     * @param end   end of range ni bytes
-     * @return
-     * @throws IOException
-     */
-    @Deprecated
-    public InputStream openInputStreamForRange(long start, long end) throws IOException {
-
-        HttpURLConnection connection = openConnection();
-        String byteRange = "bytes=" + start + "-" + end;
-        connection.setRequestProperty("Range", byteRange);
-        return new WrapperInputStream(connection, connection.getInputStream());
-    }
```

Comment:
Sorry for the word "Unused". I usually check GATK and Picard if I add a PR to remove deprecated methods/classes.